data/generated-langdb.json

```diff
@@ -316,14 +316,10 @@
             [
                 "AS"
             ],
-            "Batak Toba"
+            "ᯅᯖᯂ᯲ ᯖᯬᯅ"
         ],
         "bbc": [
-            "Latn",
-            [
-                "AS"
-            ],
-            "Batak Toba"
+            "bbc-latn"
```

Comment:
It's written in Latin, so it's self-explanatory. That's what we do for other names, like ug.